SQLQuery_ZooManagement.sql

```diff
@@ -0,0 +1,164 @@
+CREATE Database ZooManagement;
+
+Use ZooManagement;
+
+CREATE TABLE Species
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Species VARCHAR(150),
+SpeciesNumber INT,
+AnimalsNumber INT,
+);
+
+CREATE TABLE Visitors
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Day VARCHAR(150),
+NumberOfVisitors INT,
+);
+
+CREATE TABLE Tickets
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Price INT,
+Validity VARCHAR(150),
+);
+
+CREATE TABLE Caretakers
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Name VARCHAR(150),
+);
+
+CREATE TABLE ZooMentenance
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+AreaOfWork VARCHAR(150),
+);
+
+CREATE TABLE WorkDistribution

```

Comment:
Also the WorkDistribution table, if I understand the things correctly, should link the caretakers and their work areas so it's modelling a many to many relationship. It means that the table could be named Caretakers_ZoosMaintenance.